.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,41 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/ubuntu
+{
+	"name": "libchewing-devcontainer",
+	// Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
+	"build": {
+		"dockerfile": "./Dockerfile"
+	},
+	"customizations": {
+		"vscode": {
+			"extensions": [
+				"eamodio.gitlens", // IDE Git information
+				"shardulm94.trailing-spaces", // Show trailing spaces
+				"christian-kohler.path-intellisense",
+				"ms-azuretools.vscode-docker",
+				"ms-vscode.cmake-tools", // Cmake
+				// Rust language support
+				"rust-lang.rust", // Use "rust-lang.rust-analyzer" once MSRV reaches 1.64
+				// Ref:
+				// + https://github.com/chewing/libchewing/pull/347#issuecomment-1412022376
+				// + https://github.com/rust-lang/rust/blob/master/RELEASES.md#version-1640-2022-09-22
+				"tamasfe.even-better-toml", // for Cargo.toml
+				// Other
+				"genieai.chatgpt-vscode"
```

Comment:
I feel this should not be in the default extensions set. Users can install additional extensions if they want.

Reply:
25208bfe